algorithms/unit_tests/TestStdAlgorithmsCommon.hpp

```diff
@@ -199,7 +199,7 @@ auto create_deep_copyable_compatible_view_with_same_extent(ViewType view) {
   // this is needed for intel to avoid
   // error #1011: missing return statement at end of non-void function
 #if defined KOKKOS_COMPILER_INTEL ||                                  \
-    (defined(KOKKOS_COMPILER_NVCC) && KOKKOS_COMPILER_NVCC >= 1130 && \
+    (defined(KOKKOS_COMPILER_NVCC) && KOKKOS_COMPILER_NVCC >= 1100 && \
```

Comment:
I guess where I changed this, it's host code so we get `__builtin_unreachable` from the host compiler (`g++-8` in this case).

The link you mention is when `nvcc` introduces `__builtin_unreachable` for *device code*, right ?

Reply:
So, my first attempt was to solve the errors by making changes.

Then came this issue that `__builtin_unreachable` is not available for device code for `nvcc < 11.3`. So I was stuck.

So I added `--diag_suppress=940` for this only place where I was stuck (because I couldn't use `__builtin_unreachable` in a device function).

So you're right that because of this `diag_suppress`, I should be able to revert other changes. Let's do it and see.

Reply:
Seems fine.